### PR TITLE
Unix sockets should be considered local with Postgres

### PR DESCRIPTION
**Pull request by faucct**
https://github.com/DatabaseCleaner/database_cleaner/issues/561

### CODE REVIEW

**Comment by faucct** [comment]
ping

File: lib/database_cleaner/safeguard.rb [hunk 1]
```diff
@@ -52,7 +50,13 @@ def given?
         end
 
         def remote?(url)
-          url && !LOCAL.any? { |str| url.include?(str) }
+          url &&
+            case URI(url).host
+            when nil, 'localhost', /\.local/, '127.0.0.1'
```
Comment (etagwerker) [inline lib/database_cleaner/safeguard.rb]: @faucct Could you keep this as a constant? 

Also, it seems that you missed `sqlite3:` -- was that on purpose?

Comment (faucct) [reply lib/database_cleaner/safeguard.rb]: ping

**Comment by ivanbrennan** [comment]
It looks like this has been addressed in a different PR: https://github.com/DatabaseCleaner/database_cleaner/pull/569

**Pull request closed without merging**